Examples/Integration/IntegrationUITests/EnumTests.swift

```diff
@@ -24,10 +24,8 @@ final class EnumTests: BaseIntegrationTests {
       StoreOf<BasicsView.Feature?>.init
       StoreOf<BasicsView.Feature?>.init
       StoreOf<BasicsView.Feature?>.init
-      StoreOf<BasicsView.Feature?>.init
```

Comment:
These deletions in our integration suite represent scoped stores that are able to be reused rather than discarded and recreated each render.